src/render/DmnPropertiesPanelRenderer.js

```diff
@@ -46,7 +46,11 @@ export default class DmnPropertiesPanelRenderer {
       this._render(element);
     });
 
-    eventBus.on('drillDown.click', (event) => {
+    eventBus.on('detach', (event) => {
+      this.detach();
+    });
+
+    dmnjs.on('detach', () => {
```

Comment:
Are these two different events that are fired?

Reply:
Yes. The event on event bus is fired when the viewer is changed while the one on dmn-js is fired when the manager is detached.